progconcurrente/concurrente_teoria/index.html

```diff
@@ -1741,22 +1741,104 @@ <h2>JCSP (Java communicating sequential processes)</h2>
 
             <h1>
               <a id="aplicaciones" class="anchor" href="#aplicaciones" aria-hidden="true"><span aria-hidden="true"
-                  class="octicon octicon-link"></span></a>Aplicaciones
+                  class="octicon octicon-link"></span></a>Aplicaciones del paradigma
             </h1>
 
-            <ol>
-              <li>Servidores Web</li>
-              <li>Sistema multimedia</li>
-              <li>Calculo numerico</li>
-              <li>Interaccion por GUI</li>
-              <li>Sistemas gestores de bases de datos</li>
-              <li>Sistemas Operativos</li>
-              <li>Sistemas de control</li>
-              <li>Simulación</li>
-              <li>Videojuegos</li>
-            </ol>
-
-            <p>En estos ejemplos, utilizamos la programación concurrente para :<br />
+           
+            
+              <p>
+                En la cotidianeidad encontramos muchos ejemplos de concurrencia y paralelismo en múltiples procesos con los que interactuamos continuamente, de ellos podemos abstraer características propias que los identifican, así mismo como las partes que los conforman con el fin de tratar de entender y modelarlos en un entorno virtual.
+            
+              </p>
+            
+              <center><img src="images/Aplicaciones-1.jpg" width="80%" height="80%" alt=""></center>
+            
+              <p>
+                El mundo real contiene actores que ejecutan acciones independientemente, pero se comunican entre sí al realizarlas, es por esto que al modelar el mundo, se deben componer y coordinar muchas ejecuciones paralelas.
+              </p>
+            
+              <ul>
+                <li>Redes ferroviarias</li>
+                <li>El cuerpo humano y sus diferentes sistemas</li>
+                <li>Sistemas de ensamblaje dentro de fábricas</li>
+                <li>Equipos de desarrollo de software</li>
+              </ul>
+            
+              <p>Entre muchas otras acciones cotidianas que deben de coordinar distintos procesos para poder ser cumplidas con éxito y funcionar de manera óptima.
+              </p>
+            
+              <p>La concurrencia y el paralelismo nos brinda bastantes herramientas para que la modelación de dichos procesos se realice de manera más apegada a la realidad, y así mismo, encontrar formas de que estos procesos sean más eficientes, al poder separarlos y distribuirlos en pequeños sub procesos que los conformen y se ejecuten paralelamente, de una manera ligeramente más natural que secuencialmente.
```

Comment:
```suggestion
              <p>La concurrencia y el paralelismo nos brindan bastantes herramientas para que la modelación de dichos procesos se realice de manera más apegada a la realidad y, así mismo, para encontrar formas de que estos procesos sean más eficientes al poder separarlos y distribuirlos en pequeños subprocesos que los conformen y se ejecuten en paralelo, es decir, de una manera ligeramente más natural que una ejecución secuencial.
```
En la última oración se eliminaron las cacofonías causadas por los adverbios terminados en -mente.